service/cloudconfig/cloud_config.go

```diff
@@ -2,7 +2,7 @@ package cloudconfig
 
 import (
 	"github.com/giantswarm/certificatetpr"
-	k8scloudconfig "github.com/giantswarm/k8scloudconfig"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_0_1_0"
```

Comment:
Versioning is still not supported. I'm aware of that. This is on TODO list here https://github.com/giantswarm/giantswarm/issues/1384

We need some changes from latest, and we have bigger problems now. After having MVP we can sort that. Now it isn't even possible to test that.